client/src/store/new-request-module.ts

```diff
@@ -2187,17 +2183,21 @@ export class NewRequestModule extends VuexModule {
       const requestData = nr && await this.addRequestActionComment(nr)
 
       const response = requestData && await axios.patch(`/namerequests/${nrId}/edit`, requestData, {
-        headers: {
-          'Content-Type': 'application/json'
-        }
+        headers: { 'Content-Type': 'application/json' }
       })
 
-      this.mutateNameRequest(response.data)
```

Comment:
This was causing "patchNameRequests() = TypeError: null is not an object (evaluating 'response.data')."